translate.js

```diff
@@ -148,7 +148,41 @@ function translateJsonCompilerOutput (output) {
   return ret;
 }
 
+function formatAssemblyText (asm, prefix, source) {
+  if (typeof asm === typeof '' || asm === null || asm === undefined) {
+    return prefix + asm + '\n';
```

Comment:
This will print `undefined` - do we want that?

Reply:
No. This was a verbatim copy from browser-solidity. I guess it should be `prefix + (asm || '') + '\n'`